### DISCUSSION TITLE
Connect to Netbox from Excel using API?

### DISCUSSION THREAD

**Discussion by Sislik** [netbox-community/netbox#11320] [Q&A]
Hi,

as continue to https://groups.google.com/g/netbox-discuss/c/RZTVChA1RcU

did anyone managed to get the excel connect to netbox API make it working? For example with demo.netbox.dev?

When i use in PowerBI and set the connection to Anonymous:

let
    Origin = Json.Document(Web.Contents("https://demo.netbox.dev/api/virtualization/cluster-groups/", [Headers=[Authorization="bb3e7d49cc37d30ba342e005d113713a3f4f961a"]])),

I will just get Access to the resource is forbidden.

**Comment by jeremystretch** [comment]
I can't speak to how well this idea would work in general, but the authorization header should read `Token bb3e7d49cc37d30ba342e005d113713a3f4f961a`. That's likely the cause of the access issue.

**Reply by Sislik** [reply]
Thx, it is :)

**Reply by pkov-cloud** [reply]
Hi it seems I have the same task and wonder the correct syntax?
let
Origin = Json.Document(Web.Contents("https://demo.netbox.dev/api/virtualization/cluster-groups/", [Headers=[Authorization=**"Token** bb3e7d49cc37d30ba342e005d113713a3f4f961a"]]))

Thanks

**Reply by sdktr** [reply]
No, you need to drop the 
Authorization= 
part of your header.

**Reply by pkov-cloud** [reply]
> No, you need to drop the Authorization= part of your header.

Thanks
Here might be different syntax in Excel and BI, tried many options in Excel.

I have just verified with Excel
= Json.Document(Web.Contents("https://demo.netbox.dev/api/virtualization/cluster-groups/", [Headers=[Authorization="Token MyToken"]])) - is only way it works...

**Reply by sdktr** [reply]
You're right, the header key and value are both in there, sorry for the confusion
